c2corg_ui/static/js/documentdetailsview.js

```diff
@@ -9,23 +9,20 @@ goog.require('app');
 app.viewDetailsDirective = function() {
   return {
     restrict: 'A',
-    controller: 'AppViewDetailsController',
-    controllerAs: 'detailsCtrl',
-    bindToController: true,
+    controller: 'AppViewDetailsController as detailsCtrl',
+    bindToController: {'routes' : '=', 'waypoints': '=', 'orientations': '@'},
```

Comment:
How will the other document types be handled?
Do we really need to explicitly pass routes and waypoints? Can't we have something more generic?